olive/model.py

```diff
@@ -207,11 +207,12 @@ class HFComponent(ConfigBase):
 
 
 class HFConfig(ConfigBase):
-    model_name: str
+    model_name: str = None
     task: str = None
     # TODO: remove model_class and only use task
     model_class: str = None
     use_ort_implementation: bool = False
+    load_model_from_hub: bool = True
```

Comment:
I think a validator, that checks that `model_name` is not `None` if `load_model_from_hub` is `True` would be helpful.

Reply:
Will add.